clients/src/main/java/org/apache/kafka/common/requests/CreateAclsResponse.java

```diff
@@ -43,7 +43,8 @@
                     ERROR_MESSAGE))));
 
     /**
-     * The version number is bumped to indicate that on quota violation brokers send out responses before throttling.
+     * The version number is bumped to indicate that, on quota violation, brokers send out responses before throttling
+     * and request has additional resource_name_type field.
```

Comment:
It's not necessary to duplicate the comment about CreateAclsRequest inside CreateAclsResponse.  The response doesn't have any additional fields.  I don't think there's anything to document here beyond the original comment about quota stuff.

Reply:
done.